zh/_theme/phalcon/layout.html

```diff
@@ -153,7 +153,7 @@
       };
     </script>
 
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
+    <script src="http://apps.bdimg.com/libs/jquery/1.8.1/jquery.min.js"></script>
```

Comment:
googleapis.com is very slowly in China . the new url is very fast in China. baidu.com support this cdn .  baidu is the bigest search engine like google in china .